### DISCUSSION TITLE
Since upate TTF35 can't print from TTF menu

### DISCUSSION THREAD

**Discussion by rmaltais** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2115] [General]
hi,
Since new update, all command work fine but no print from any card or internal. I can select and i hear a beep when i select file, but no temp and no move from printer. When i select file the screen stay there with no action from printer, just a beep.

I can do a ABL, move axis, adjust temp and all other functions but no print.

My screen work very fine before this update BIGTREE_TFT35_V3.0_E3.27.x.bin

thank

**Comment by kisslorand** [comment]
Check #2109 and #2103.

**Reply by rmaltais** [reply]
Thanks a lot for this reply.

**Reply by rmaltais** [reply]
This fix my problem. Thank a lot.
